main/openrc/modloop.initd

```diff
@@ -99,7 +99,35 @@ start() {
 		mount -t overlay -o upperdir=/.modoverlayfs/modules,lowerdir=/.modloop/modules,workdir=/.modoverlayfs/work overlay /lib/modules
 		eend $? || return 1
 	else
-		rm -rf /lib/modules && ln -sf /.modloop/modules /lib/
+		# if we have a directory /lib/modules/$(uname -r)/local with modules, preserve these
+		LOCAL_MODULE_DIR=/lib/modules/$(uname -r)/local
+		if [ -z $KOPT_local_modules ]; then
+			if [ ! -d $LOCAL_MODULE_DIR ]; then
+				mkdir $LOCAL_MODULE_DIR
+			fi
+			ebegin "Combining local kernel modules with /.modloop/modules"
+			# move local modules dir to tmp
+			mv $LOCAL_MODULE_DIR /tmp
+			# remove all modules and link all directories from /.modloop/modules/$(uname -r)/
+			rm -rf /lib/modules && mkdir -p /lib/modules/$(uname -r)
```

Comment:
Please don't delete `/lib/modules`, since it might contain package-supplied files.